Makefile

```diff
@@ -48,7 +48,7 @@ export PLUGIN_SUBMODULE
         test-vm test-rest-vm test-all-vm test-bchain test-rest-bchain test-all-bchain                                            \
         test-prove test-failing-prove                                                                                            \
         test-prove-benchmarks test-prove-functional test-prove-opcodes test-prove-erc20 test-prove-bihu test-prove-examples      \
-        test-prove-mcd test-klab-prove test-haskell-dry-run                                                                      \
+        test-prove-mcd test-klab-prove                                                                                           \
```

Comment:
I wanted this option. Any reason you want to remove it?
It makes it very convenient to test the kprove front end and make improvements.

Reply:
Well you can recover this by just calling `make test-prove KPROVE_OPTS=--dry-run`